src/types/Options.ts

```diff
@@ -1,5 +1,6 @@
 export default interface Options {
     project?: string;
     rootDir?: string;
+    ignoreNodeModules?: boolean;
```

Comment:
Can we call this `ignoreExternalFences`?

```suggestion
    ignoreExternalFences?: boolean;
```